src/js/components/ResortHeader/ResortHeader.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {
+  CCard,
+} from '@coreui/react';
+import PropTypes from 'prop-types';
+
+const ResortHeader = (props) => {
+  const { resortInfo: { title, location: { country, state } } } = props;
+  return (
+    <CCard className="resort-header__card p-4">
+      <h1 className="display-5 text-center resort-title" color="secondary">{title}</h1>
+      <div className="text-center mb-2">
+        <span className="state-name">{state && state.name ? state.name + ' ' | ' ' : ''}</span>
```

Comment:
```suggestion
        {state && state.name &&
            <span className="state-name">state.name</span>
        }
```

https://reactjs.org/docs/conditional-rendering.html

Reply:
Also, as I mentioned on slack. The logic in `state && state.name` is a bit weak. The reason is, it's only checking is `state` is truthy. If `state` was a string, this code would break the app. So it's lucky we have `propTypes` protecting us, though that's not very well declared by this logic.

Reply:
I didnt realise you can write syntax like this.